server/services/twilio/twilio.main.js

```diff
@@ -8,18 +8,19 @@ function send( to, from, body, attempts ) {
         to: '+1' + to,
         from: '+1' + from,
         body: body
-    }, function( err, data ){
-        if( err ) {
+    }, function( error, data ){
+        if (error) {
+            log.error('Sending Monthly Options text using twilio falied', {error}, {to, from, message: body, resendAttempts: attempts} );
```

Comment:
Oh, that's ES6, which I've been trying to practice, but is optional at this point. If you feel strongly about using it or not, I'm fine with going either way. 
```
const x = 2
function z () { return false }
var es5 = {
  x: x,
  y: function (n) { return Math.random() > n }
  z: z
}
let es6 = {
  x, 
  y: function (n) { return Math.random > n },
  z
}
```